src/main/java/org/terasology/potions/PotionCommonEffects.java

```diff
@@ -44,6 +44,7 @@
     public static final String FEATHER_FALL = "FEATHER_FALL";
     public static final String BRICK_FALL = "BRICK_FALL";
     public static final String MULTI_JUMP = "MULTI_JUMP";
+	public static final String NO_VISIBILITY = "NO_VISIBILITY";
```

Comment:
Could you add something above this to indicate the pending effect implementation? Like:

`// TODO: Effect implementation awaiting API improvements, such as https://github.com/MovingBlocks/Terasology/pull/2449